sympy/holonomic/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""
+The Holonomic module is intended to deal with Holonomic Functions along
```

Comment:
I think we could have 'holonomic' etc. all in lower case in this text. There is also a directive `:mod:` to indicate module names.